app/controllers/api/v1/search_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Api
+  module V1
+    class SearchController < Api::V1::BaseApiController
+      include SearchDoc
+
+      skip_before_filter :authenticate_from_token!
+
+      def create
+        query_string = params[:query]
+        repository = Elasticsearch::Persistence::Repository.new
+        repository.index = 'checkdesk_application_development_annotations'
+        repository.type = 'annotation'
+        result = repository.search(query: { query_string: {query: query_string}})
```

Comment:
Do you need to re-instantiate the repository? I was hoping that you could use `Annotation.search`, as we do on annotation_base